lib/m2ts/m2ts.js

```diff
@@ -33,6 +32,9 @@ var
 TransportPacketStream = function() {
   var
     buffer = new Uint8Array(MP2T_PACKET_LENGTH),
+    totalDataEmitted = 0,
+    sinceFlushDataEmitted = 0,
+    invalidPushCalls = 0,
```

Comment:
as of right now there is not plan to do anything with the stats outside of checking for errors